sopel/modules/url.py

```diff
@@ -186,16 +186,12 @@ def find_title(url, verify=True):
     """Return the title for the given URL."""
     response = requests.get(url, stream=True, verify=verify)
     try:
```

Comment:
just curious: is the try/finally still necessary, or could this just be simplified a bit further?